projects/distributed-tracing/src/shared/dashboard/data/graphql/graphql-data-source.model.ts

```diff
@@ -53,7 +53,6 @@ export abstract class GraphQlDataSourceModel<TData> implements DataSource<TData>
 
     this.querySubject.next({
       buildRequest: this.convertToBuilder(requestOrBuilder),
-      isolated: true,
```

Comment:
This does mean we'll be batching certain things that we hadn't batched before we removed batching the first time - such as topology, which has always been isolated.

Reply:
and this would be fine wouldn't it?

Reply:
Those were initially meant for queries known to be expensive, which is why we unrolled the batching in the first place. The flow for example requires multiple lower level queries so is expected to be slower.

Reply:
We can have query function accept the `isolated` property or may be the entire request option that includes cacheability. Then root data source would add it to RequestEvent and emit. Query event service would then pass it on to graphql request service. This would give more configuration options for us at data source level.

Reply:
how does this sound?

Reply:
Sounds right I think.

Reply:
Done. We can parameterize cacheability and isolated properties on root ds. For now, I am only keeping it as function param.